### ISSUE TITLE
Add QMK firmware

### ISSUE THREAD

**Issue by filoxo** [cozykeys/bloomer#9]
Hey, thanks for this great design! A buddy of mine purchased a few of these and asked me to create a version of QMK for this. I can do that, but I'd like to check in with you and see if you already have the firmware available somewhere. QMK/TMK or any other firmware would be most helpful so I don't have to read the schematics instead. 

If you don't have something you can share, no worries, I'll see what I can accomplish and share if/when its done. Thanks again!

**Comment by pcewing** [comment]
Hey! Thanks for the reminder; I actually completely forgot to push it to the
QMK repository. I went ahead and pushed what I have to my fork here:

https://github.com/pcewing/qmk_firmware/tree/bloomer

I just tried to build to make sure it's in a good state but I'm seeing an error
that the firmware is too large. I don't have time to investigate right now but
it still might be easier to pick up from there than start from scratch. 

If I can't get around to it tonight I'll try to figure it out this weekend.
Hope that helps!
